moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -647,6 +647,10 @@ class KinematicsBase
     return false;
   }
 
+  void storeValues(const moveit::core::RobotModel& robot_model, const std::string& group_name,
```

Comment:
add comment - what does this function do? what is it storing where?

Reply:
Will add a comment. It's a replacement for `setValues()`.